core/trace/dt_span_enricher.go

```diff
@@ -72,3 +75,48 @@ func fw4TagFromContextOrMetadata(ctx context.Context) *fw4.Fw4Tag {
 	}
 	return nil
 }
+
+var emptyMapValue struct{}
```

Comment:
Why not bool true?

Reply:
an empty struct is more efficient since it has 0 bytes